cypress-io/cypress
# feat(unify): cypress version update ui

- Closes https://cypress-io.atlassian.net/browse/UNIFY-382

### User facing changelog

This adds a modal that appears when a user clicks the "Upgrade" button and contains instructions to upgrade, and removes the dropdown showing intervening versions when you are on latest. Now, if you are on latest, this is displayed.

### Additional details

This PR:

- updates the `dev` command for frontend-shared to match the others
- moves & renames the inline code fragment component from launchpad to shared
- updates tests & topnav component
- adds an UpdateCypressModal component + CT test

### How has the user experience changed?

Previously the "update" button was not wired up to do anything. Now it triggers this modal:

<img width="752" alt="Screen Shot 2021-11-19 at 9 15 54 AM" src="https://user-images.githubusercontent.com/8340719/142636985-f13ded27-9589-4560-b2dd-98d2fbd2c353.png">

👆 This is the final version of this, after review with Ryan

Video:

https://user-images.githubusercontent.com/8340719/142246522-2fde5180-63e6-497d-8607-822370698bc5.mov

### How to test

Via tests - Run the HeaderBarContent component tests and see that the prompt is present when the latest published version of cypress is not the same as the currently installed version.

If you want to use the the running app - first run `yarn cypress:open` and see no prompt to upgrade, as we are on 9.0.0. Downgrade cypress in the root cypress folder, run `yarn` and `yarn cypress:open` and see that when we are downgraded, the prompt and modal are working

### PR Tasks
